index.html

```diff
@@ -361,7 +361,13 @@ <h2><code>PointerEvent</code> Interface</h2>
                         </dd>
                     <dt><dfn>altitudeAngle</dfn></dt>
                         <dd>
-                            <p>The altitude (in radians) of the transducer (e.g. pen stylus), in the range [0,π/2] - where 0 is parallel to the surface (X-Y plane), and π/2 is perpendicular to the surface. For hardware and platforms that do not report tilt or angle, the value MUST be 0.</p>
+                            <p>The altitude (in radians) of the transducer (e.g. pen stylus), in the range [0,π/2] - where 0 is parallel to the surface (X-Y plane), and π/2 is perpendicular to the surface. For hardware and platforms that do not report tilt or angle, the value MUST be π/2.</p>
+                            <div class="note">
+                                When the hardware or platform does not report the tilt or angle, as opposed to the default value of 0 defined for <code>altitudeAngle</code>
+                                in <a href="https://w3c.github.io/touch-events/">Touch Events - Level 2</a> specification,
```

Comment:
Just an editorial comment and not important really. You can also reference the touch events with brackets similar to other places in this spec so that it appear in the normative references.

Reply:
I checked the [reference database](https://www.specref.org/?q=touch%20events) and I there is no entry for Touch Events - Level 2. I believe this is why it is referenced like this here.